src/feature/control/control_getinfo.c

```diff
@@ -325,6 +325,33 @@ getinfo_helper_current_time(control_connection_t *control_conn,
   return 0;
 }
 
+/**
+ * Helps to access directory information for a given
+ * type of consensus.
+ *
+ * returns: int
```

Comment:
`returns: int` is redundant, because the code tells us the type that the function returns.